### PR TITLE
Documentation update for Az.Support module

### PR DESCRIPTION
**Pull request by shahbj79**
## Description

Updated documentation of Az.Support module with additional example for New-AzSupportTicket cmdlet.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Support/Support/Az.Support.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.0'
+ModuleVersion = '1.0.1'
```
Comment (VeryEarly) [inline src/Support/Support/Az.Support.psd1:15]: please do not change version, Azure-PowerShell team will bump it during each release

File: src/Support/Support/ChangeLog.md [hunk 2]
```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 1.0.1
```
Comment (VeryEarly) [inline src/Support/Support/ChangeLog.md:23]: please do not change version, Azure-PowerShell team will bump it during each release

File: src/Support/Support/ChangeLog.md [hunk 3]
```diff
@@ -20,6 +20,9 @@
...
+* Documentation update of `Az.Support` module
```
Comment (VeryEarly) [inline src/Support/Support/ChangeLog.md:24]: no release note needed for document update

**Pull request merged**